docs/DeveloperGuide.adoc

```diff
@@ -18,8 +18,13 @@ By: `CS2103T-T13-1`      Since: `Sep 2019`      Licence: `MIT`
 
 == Introduction
 Welcome to ORGANice! This is an organ matching desktop application that is specifically designed for fast typing
-hospital administration staff. This developer guide is meant for developers who would like to understand the
-design and structure of ORGANice.
+hospital administration staff.
+
+=== Purpose
+This document is written to document the software design and and decisions made in the implementation of ORGANice.
```

Comment:
This guide documents the software design and considerations made during the development of ORGANice.